docs/manual/usage.rst

```diff
@@ -105,47 +88,11 @@ In some cases this is not desired. To make the output of the simulator determini
     qxelarator.execute_string("version 1.0;qubits 2;h q[0];measure_all", iterations=1000, seed=123)
 
 
-JSON output
```

Comment:
Do you know if Pablo changed the output format of QX? This needs to align with the expectations of the QI platform.

Reply:
Right, I see lots of code have to do with the removal of a command line option to output results as JSON. And I have no idea why Pablo decided to do this. We could ask him.

Reply:
We've got an answer to this from Pablo:

> The JSON output was useful when there was no API from python to retrieve the different pieces of data in the results.
> It was a request from QI but then I created the new API and they have integrated it.
Maarten had confirmed that they no longer use the JSON and that they were happy with the new API.
> It’s no longer useful as there is now possibility to get everything directly from Python.
> This should be in the docs